kivy_garden/matplotlib/backend_kivy.py

```diff
@@ -1321,22 +1302,85 @@ def _on_mouse_pos(self, *args):
         y = newcoord[1]
         inside = self.collide_point(*pos)
         if inside:
-            self.motion_notify_event(x, y, guiEvent=None)
+            self.motion_notify_event(x, y)
         if not inside and not self.entered_figure:
-            self.leave_notify_event(guiEvent=None)
+            self.leave_notify_event()
             self.entered_figure = True
         elif inside and self.entered_figure:
-            self.enter_notify_event(guiEvent=None, xy=(pos[0], pos[1]))
+            self.enter_notify_event()
             self.entered_figure = False
 
-    def enter_notify_event(self, guiEvent=None, xy=None):
-        event = Event("figure_enter_event", self, guiEvent)
+    def enter_notify_event(self, gui_event=None):
```

Comment:
These changes are making the API backward-incompatible, but since we're moving to a new package, that's also an opportunity to clean up bad things.

Reply:
Well spotted. Didn't think of that, thanks for the hint.